python/runtime/dbapi/hive_test.py

```diff
@@ -51,15 +51,15 @@ def test_query(self):
 
     def test_exec(self):
         conn = HiveConnection(testing.get_datasource())
-        rs = conn.exec("create table test_exec(a int)")
+        rs = conn.execute("create table test_exec(a int)")
```

Comment:
Is the test running on the CI currently?

Reply:
Not really, I will add it.